index.test.js

```diff
@@ -110,18 +194,28 @@ test('fastify.mail.sendMail calls nodemailer.sendMail with correct arguments', a
 
   const sendMailStub = sinon.stub(fastify.nodemailer, 'sendMail')
 
-  await fastify.mail.sendMail(testRecipients, relative(__dirname, testTemplates), testContext)
+  await fastify.mail.sendMail(testMessage, { templatePath: relative(__dirname, testTemplates), context: testContext })
 
-  t.same(sendMailStub.args[0], [testMessage])
+  t.same(sendMailStub.args[0], [responseWhenTemplatesPresent])
   t.is(sendMailStub.args.length, 1)
 })
 
-test('fastify.mail.sendMail returns an error with missing arguments', async (t) => {
+test('fastify.mail.sendMail returns an error when required values are not present (from)', async (t) => {
+  t.teardown(() => fastify.close())
+  const fastify = Fastify()
+  fastify.register(fastifyMail, { pov: { engine: { nunjucks } }, transporter: { jsonTransport: true } })
+  await fastify.ready()
+  const { error } = await fastify.mail.sendMail({ to: 'to@ignoreme.com', subject: 'subject@ignoreme.com' })
+  t.ok(error instanceof TypeError)
+  t.is(error.message, '"from" must be defined')
+})
+
+test('fastify.mail.sendMail returns an error when no message is defined', async (t) => {
   t.teardown(() => fastify.close())
   const fastify = Fastify()
   fastify.register(fastifyMail, { pov: { engine: { nunjucks } }, transporter: { jsonTransport: true } })
   await fastify.ready()
   const { error } = await fastify.mail.sendMail()
   t.ok(error instanceof TypeError)
```

Comment:
newline above our asserts here in order to call them out better visually

Reply:
done